src/ucc/api/ucc_status.h

```diff
@@ -39,6 +39,7 @@ typedef enum {
     UCC_ERR_NO_RESOURCE                 =   -5,
     UCC_ERR_NO_MESSAGE                  =   -6, /*!< General purpose return code without specific error */
     UCC_ERR_NOT_FOUND                   =   -7,
+    UCC_ERR_TIMED_OUT                   =   -8,
```

Comment:
Should we make this an error or status? It can still run to completion correctly, right?

Reply:
do you mean making it >0 ? maybe. @Sergei-Lebedev @bureddy  wdyt?

Reply:
it should be error right? user does not care and check if coll is completed or not after timeout

Reply:
Typically with error, if we detect library doesn't continue the execution. In this case, we are continuing the execution.